README.md

```diff
@@ -101,11 +101,11 @@ var handleMessage = (state, message) => {
 
 ```js
 var handleAction = (action, sbot, callback) => {
-  var content = {
+  var newMessage = {
```

Comment:
`newMessage` might be a bit confusing, because everywhere else `message` refers to an object with `key`, `value`, `timestamp`, where `value` contains `author`, `content`, etc.

`newMessageContent`?

Reply:
https://github.com/ahdinosaur/ssb-bot/commit/0116f0550351113f21c3922dc52772f42aa96c14